synapse/models/geospace.py

```diff
@@ -475,25 +511,31 @@ def getModelDefs(self):
                     )),
 
                     ('geo:telem', {}, (
+
                         ('time', ('time', {}), {
-                            'doc': 'The time that the node was at the position.'}),
+                            'doc': 'The time that the telemetry measurements were taken.'}),
+
                         ('desc', ('str', {}), {
                             'doc': 'A description of the telemetry sample.'}),
+
```

Comment:
Do we want to add a tri:latlong prop to account for latlong values obtained via trilateration?

Reply:
The origin of the information does not make it a different field and the geo:locatable interface already accounts for being able to capture accuracy.